src/Workspaces/CoreTestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -166,7 +166,7 @@ public InProcRemoteServices(SolutionServices workspaceServices, TraceListener? t
             {
                 var remoteLogger = new TraceSource("InProcRemoteClient")
                 {
-                    Switch = { Level = SourceLevels.Verbose },
+                    Switch = { Level = SourceLevels.Off },
```

Comment:
needed to change this because the amount of cost of this logging, over the stdout pipe which xunit captures, was actually slowing down tests too much! =-o